ConventionMobile/ConventionMobile/GlobalVars.cs

```diff
@@ -4,22 +4,50 @@
 using ConventionMobile.Views;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
-//using Plugin.Toasts;
 using System;
 using System.Collections.Generic;
-using System.IO;
 using System.Linq;
-using System.Reflection;
 using System.Text;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using System.Web;
+using ConventionMobile.Business;
 using Xamarin.Forms;
 
 namespace ConventionMobile
 {
     public class GlobalVars
     {
+        public static GenEventsLoadingViewModel GenConBusiness = new GenEventsLoadingViewModel();
+        public static GenSearchView View_GenSearchView = null;
+        public static GenMapView View_GenMapView = null;
+        public static GenUserListView View_GenUserListView = null;
+        public static GenEventsLoadingView View_GenEventsLoadingView = null;
+
+        public enum ThemeColors
+        {
+            Primary = 0,
+            Secondary = 1,
+            ActionElement = 2
+        };
+
+        public readonly static List<Color> ThemeColorsBG = new List<Color>()
+        {
+            //new Color(244/255, 178/255, 52/255),
+            //new Color(252/255, 205/255, 116/255),
+            //new Color(213/255, 36/255, 36/255)
+            new Color(0.9568, 0.6980, 0.2039),
+            new Color(0.9882, 0.8039, 0.4549),
+            new Color(0.8352, 0.1412, 0.1412)
+        };
```

Comment:
we should come up with a better way to define these guys.  This is very undescriptive.  if nothing else, comment the color with the "color name"

Reply:
Yes, I totally agree. The 3 colors are meant to match the general theme of the year, and will therefore also need to be updated yearly. I don't currently have the capacity to supply color information via the web update service. We can add comments for 2019 as dark orange, light orange, and red.